packages/components/Tabs/src/TabsItem.types.ts

```diff
@@ -0,0 +1,96 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
+import { ITextProps } from '@fluentui-react-native/text';
+import { IconProps } from '@fluentui-react-native/icon';
+import { IButtonProps, IButtonTokens } from '@fluentui-react-native/button';
+
+export const tabsItemName = 'TabsItem';
+
+export interface TabsItemInfo extends IPressableState {
+  /**
+   * Disables the TabsItem.
+   * @default false
+   */
+  disabled?: boolean;
```

Comment:
Why does this need to be in the state? Why not just props?

I don't believe you need this in the state if you're not using it in the render() function.